Can be removed when upgrading
Upstream: Yes
From 2c949e0d82049234f31cd595c6d2097bc9c241de Mon Sep 17 00:00:00 2001
From: "Dmitry V. Levin" <ldv@altlinux.org>
Date: Sat, 28 Nov 2020 08:00:00 +0000
Subject: [PATCH] xlat: remove IPPROTO_MAX

* xlat/inet_protocols.in (IPPROTO_MAX): Remove.

Reported-by: Khem Raj <raj.khem@gmail.com>
Suggested-by: Eugene Syromiatnikov <esyr@redhat.com>
Resolves: https://github.com/strace/strace/issues/164
---
 xlat/inet_protocols.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/xlat/inet_protocols.in b/xlat/inet_protocols.in
index c37508fa3..89aa56ac4 100644
--- a/xlat/inet_protocols.in
+++ b/xlat/inet_protocols.in
@@ -32,5 +32,4 @@ IPPROTO_UDPLITE		136
 IPPROTO_MPLS		137
 IPPROTO_ETHERNET	143
 IPPROTO_RAW		255
-IPPROTO_MAX		256
 IPPROTO_MPTCP		262
